lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -390,8 +390,6 @@
         .ellipsis-cell {
             .cell-container {
                 height: 100%;
-                display: inline-grid;
-                width: 100%;
```

Comment:
This might break the ellipsis on the document list in the non-card layout

Reply:
this css wasn't there
https://github.com/Alfresco/alfresco-ng2-components/commit/e94ce4602bbcba3ec5fcc1bfae700e0fd773a80b#diff-316178d67dd72869800ff3be79152f40R393
If the problem is the document list, can we add this css to the document list scss ?

Reply:
<img width="1236" alt="screen shot 2018-06-18 at 10 02 57" src="https://user-images.githubusercontent.com/14923305/41527615-a1f14b1c-72df-11e8-84d7-77a50e1cfe99.png">
Because of that APW is having a regression